src/sql/workbench/browser/modal/calloutDialog.ts

```diff
@@ -3,86 +3,35 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { URI } from 'vs/base/common/uri';
 import * as TelemetryKeys from 'sql/platform/telemetry/common/telemetryKeys';
-import * as DOM from 'vs/base/browser/dom';
-import * as styler from 'vs/platform/theme/common/styler';
-import * as strings from 'vs/base/common/strings';
 import { IDialogProperties, Modal, DialogWidth } from 'sql/workbench/browser/modal/modal';
 import { IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
-import { localize } from 'vs/nls';
-import { IContextViewService } from 'vs/platform/contextview/browser/contextView';
 import { IClipboardService } from 'vs/platform/clipboard/common/clipboardService';
-import { IFileDialogService, IOpenDialogOptions } from 'vs/platform/dialogs/common/dialogs';
 import { IThemeService } from 'vs/platform/theme/common/themeService';
 import { ILogService } from 'vs/platform/log/common/log';
 import { ITextResourcePropertiesService } from 'vs/editor/common/services/textResourceConfigurationService';
 import { IAdsTelemetryService } from 'sql/platform/telemetry/common/telemetry';
-import { attachModalDialogStyler } from 'sql/workbench/common/styler';
-import { ILayoutService } from 'vs/platform/layout/browser/layoutService';
-import { Deferred } from 'sql/base/common/promise';
-import { InputBox } from 'sql/base/browser/ui/inputBox/inputBox';
-import { Checkbox } from 'sql/base/browser/ui/checkbox/checkbox';
-import { RadioButton } from 'sql/base/browser/ui/radioButton/radioButton';
-import { IPathService } from 'vs/workbench/services/path/common/pathService';
 
-export type CalloutType = 'IMAGE' | 'LINK';
+import { ILayoutService } from 'vs/platform/layout/browser/layoutService';
 
 export interface ICalloutDialogOptions {
```

Comment:
This should be moved into the superclass definitions since it's specifically for those